<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TV-sniffer-alpha</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resource.qrc">
    <normaloff>:/images/titl.png</normaloff>:/images/titl.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>23</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_shutdown"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>抓取</string>
    </property>
    <addaction name="action_config"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>50</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>100</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStartCapture"/>
   <addaction name="actionStopCapture"/>
  </widget>
  <action name="action_config">
   <property name="text">
    <string>配置</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionStartCapture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/images/capture_start_24.png</normaloff>:/images/capture_start_24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;start</string>
   </property>
   <property name="toolTip">
    <string>开始</string>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionStopCapture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/images/capture_stop_24.png</normaloff>:/images/capture_stop_24.png</iconset>
   </property>
   <property name="text">
    <string>S&amp;top</string>
   </property>
   <property name="toolTip">
    <string>停止</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_static_data">
   <property name="text">
    <string>协议</string>
   </property>
  </action>
  <action name="action_main_ui">
   <property name="text">
    <string>主界面</string>
   </property>
  </action>
  <action name="actionHost_package">
   <property name="text">
    <string>IP Statistics</string>
   </property>
  </action>
  <action name="action_export_file">
   <property name="text">
    <string>导出文件</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="action_shutdown">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
